config/default.yaml

```diff
@@ -13,6 +13,7 @@ security:
     mediaSrc: 'self'
     frameSrc: 'none'
     imgSrc: ['self', 'www.google-analytics.com', 'hmctspiwik.useconnect.co.uk']
+  roles_whitelist: "ccd-import"
```

Comment:
trying to understand why this whitelist is different from the one in main.tf

Reply:
No real reason; the one in `main.tf` has the additional role `ccd-validate` only because the ccdimportdomain@gmail.com user had this role, in addition to `ccd-import`. However, only `ccd-import` is required (for now)